Tryouts/Prototypes/ModuleLoader/ModuleLoaderPrototype/ModuleLoaderPrototype/Modules/IModule.cs

```diff
@@ -0,0 +1,12 @@
+﻿using ModuleLoaderPrototype.Interfaces;
+
+namespace ModuleLoaderPrototype.Modules;
+
+public interface IModule
```

Comment:
Are client apps expected to implement this interface? It seems they're not so I'd suggest making it (and the related types) internal.

Reply:
File structure was rearranged, although a review on visibilities sounds like a good idea